docs/dashboard/apis.md

```diff
@@ -467,9 +467,386 @@ Responses
 
 > **_404 Not Found_**
 
-```javascript
+```json
 {
     "statusCode": 400,
     "message": "Smart contract not found"
 }
 ```
+
+#### 8. Add spending limit rule to a paymaster
+
+> **_POST Request_**
+
+URL: https://paymaster-dashboard-backend.prod.biconomy.io/api/v2/public/sdk/paymaster-policy
+
+Parameters
+
+Header
+
+| Param     | Type   | Description                        | Required |
+| --------- | ------ | ---------------------------------- | -------- |
+| authToken | string | Token unique to every user account | Required |
+| apiKey    | string | API Key Associated with dApp       | Required |
+
+Body
+
+| Param      | Type   | Description                                  | Required |
+|------------|--------|----------------------------------------------| -------- |
+| name       | string | Unique Policy name                           | Required |
+| policyType | string | Pass "PAYMASTER_SA_LIMIT" for spending limit | Required |
+| policyData | object | Described below                              | Required |
+
+Policy Data
+
+| Param      | Type  | Description                                                                                                             | Required |
+|------------|-------|-------------------------------------------------------------------------------------------------------------------------| -------- |
+| type       | string | "PAYMASTER"(Global limit) / "SMART_ACCOUNT" (Individual User Limit)                                                     | Required |
+| cycleDuration | object | eg. `{value:3, unit: "hour"}`, `{value:2, unit: "day"}`. <br/> Currently only supported time units are "hour" and "day" | Required |
+| threshold | number | Limit Value                                                                                                             | Required |
```

Comment:
us the threshold type is NATIVE_ASSET then can this threshold will be in decimals?

Reply:
Yes, number here means a javascript number